packages/react-core/src/components/TextInput/__tests__/TextInput.test.tsx

```diff
@@ -124,4 +124,13 @@ describe('TextInput', () => {
     render(<TextInput isLeftTruncated aria-label="start truncated text input" />);
     expect(trimLeftFn).toHaveBeenCalled();
   });
+
+  test('has aria-expanded set to true when ariaProps.isExpanded is true', () => {
+    render(<TextInput expandedProps={{isExpanded: true, ariaControls: 'test'}}  aria-label="isExpanded"/>);
+
+    const input = screen.getByLabelText('isExpanded');
+    expect(input).toHaveAttribute('aria-expanded', 'true');
+    expect(input).toHaveAttribute('role', 'combobox');
+    expect(input).toHaveAttribute('aria-controls', 'test');
+  });
```

Comment:
I might lean towards having this split up to 3 separate tests rather than checking  that all 3 attributes are present.

For the role test, we'd also be able to do `const input = screen.getByRole('combobox')` as well, then just check that it's visible/in the document

Reply:
I don't know that it is worth it to re-render the same thing to check for different attributes?